src/components/MissionList.jsx

```diff
@@ -1,17 +1,36 @@
-import React from 'react';
-import '../styles/MissionList.css';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMission } from '../redux/missions/missionsSlice';
 import MissionCard from './MissionCard';
+import '../styles/MissionList.css';
+
+const MissionList = () => {
+  const { missionItems } = useSelector((state) => state.mission);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMission());
+  }, [dispatch]);
 
-const MissionList = () => (
-  <table border={1} className="mission-table">
-    <tr>
-      <th>Mission</th>
-      <th>Description</th>
-      <th>Status</th>
-      <th>   </th>
-    </tr>
-    <MissionCard />
-  </table>
-);
+  return (
+    <table border={1} className="mission-table">
+      <tr>
+        <th>Mission</th>
+        <th>Description</th>
+        <th>Status</th>
+        <th>   </th>
+      </tr>
+      {missionItems.map((item) => (
+        <MissionCard
+          key={item.mission_id}
+          id={item.mission_id}
+          missionName={item.mission_name}
+          missionDescription={item.description}
+        />
```

Comment:
- [ ] Here you just pass a single object(mission) as a prop:
 ``` {missionItems.map((mission) => (
        <MissionCard
          key={mission.mission_id}
          mission={mission}
        /> ```

This way your code will be clean and very readable.

Reply:
Thank you Pablo! This is very helpful to me and I will do this 💪